pkg/logs/input/docker/launcher.go

```diff
@@ -25,6 +26,9 @@ const (
 	backoffMaxDuration     = 60 * time.Second
 )
 
+// serviceFunc is separated from overrideSource for testing purpose
+var serviceFunc = input.ServiceFromTags
```

Comment:
Should we add it part of the `Launcher` struct ?

Reply:
yes done 👍 (same for the kubernetes launcher)